.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "rimage"]
-	path = rimage
-	url = ../rimage.git
+        path = rimage
+        url = https://github.com/mrajwa/rimage.git
+        branch = fix_magic_id
```

Comment:
This is not required: you don't need any `.gitmodules` change, not even temporarily for testing. github can find any SHA1 in any fork, see latest rimage README and the demos / evidence linked from https://github.com/thesofproject/rimage/pull/71

Reply:
> github can find any SHA1 in any fork,

You do need to push the referenced rimage SHA1 to any branch on github though, github cannot fetch SHA1s from  your workstation :-)

https://github.com/thesofproject/sof/pull/4790/checks?check_run_id=3667029487
```
   Submodule 'rimage' (https://github.com/thesofproject/rimage.git) registered for path 'rimage'
  Cloning into '/home/runner/work/sof/sof/rimage'...
  Error: fatal: remote error: upload-pack: not our ref a34b5dc15b638496adcfe59ca6a3afd0af0dc020
  Error: fatal: Fetched in submodule path 'rimage', but it did not contain a34b5dc15b638496adcfe59ca6a3afd0af0dc020. Direct fetching of that commit failed.
```

Reply:
Hmm so it doesn't look for any rimage fork as if that is the case it would find https://github.com/mrajwa/rimage/commit/a34b5dc15b638496adcfe59ca6a3afd0af0dc020 no?

Reply:
Very interesting, you just proved something I was wondering about: you did NOT create your rimage clone as a clone of https://github.com/thesofproject/rimage, it is "detached" and github does not magically infer the relationship. Have a look at the top-left of this page https://github.com/marc-hb/rimage and compare with yours. Your rimage misses the text "forked from thesofproject/rimage". I need to add this to the new README.md

https://stackoverflow.com/questions/36225381/reattach-a-detached-fork-in-github

(cc: @mbolivar-nordic)